packages/webpack/index.js

```diff
@@ -9,10 +9,10 @@ let convertConfig = require('./convert-config')
 let runWebpack = require('./run-webpack')
 let getConfig = require('./get-config')
 
-const WEBPACK_EMPTY_PROJECT = 962
-const WEBPACK_EMPTY_PROJECT_GZIP = 461
-const WEBPACK_EMPTY_PROJECT_IMPORT = 965
-const WEBPACK_EMPTY_PROJECT_IMPORT_GZIP = 473
+const WEBPACK_EMPTY_PROJECT = 0
```

Comment:
You need to fix these constants according to the new webpack loader in the bundle (code which webpack adds in additional to user’s code).

We are using them to subtract webpack’s own code from the bundle to calculate the cost of added library:

```
libraryCost = projectWithLib - emptyProject
```

Change the value to have `0` in empty project test https://github.com/ai/size-limit/blob/main/packages/size-limit/test/run.test.js#L300-L306

And change `IMPORT` constants to have `1` on importing the single digit https://github.com/ai/size-limit/blob/main/packages/webpack/test/index.test.js#L240-L258